### PR TITLE
Remove go.rice in favour of go generate

### PR DESCRIPTION
**Pull request by adambabik**
As fetch.go was the only consumer of go.rice, it could be safely removed in favour of go-bindata and go generate.

Motivation to do that was the fact that `go.rice` throws panic in iOS when compiled with gomobile.

### CODE REVIEW

File: geth/jail/internal/fetch/Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
-dist-fetch.rice-box.go: dist-fetch/bundle.js
-	rice embed-go
+generate: dist-fetch/bundle.js
+	go generate .
+	gofmt -s -w dist_fetch.go
```
Comment (divan) [inline geth/jail/internal/fetch/Makefile]: Not sure simplify option (`-s`) has any benefit here for autogenerated code.

File: geth/jail/internal/fetch/Makefile [hunk 2]
```diff
@@ -1,5 +1,6 @@
-dist-fetch.rice-box.go: dist-fetch/bundle.js
-	rice embed-go
+generate: dist-fetch/bundle.js
+	go generate .
```
Comment (divan) [inline geth/jail/internal/fetch/Makefile]: I don't see any `//go:generate` comments in code.

Comment (adambabik) [reply geth/jail/internal/fetch/Makefile]: There is here https://github.com/status-im/status-go/pull/444/files#diff-85d1740e9cc6fe333147d0feb6046375R3

File: geth/jail/internal/fetch/Makefile (hunk 1 above)
Comment (adambabik) [reply geth/jail/internal/fetch/Makefile]: Might not but I don't see any negative impact having `-s`.

File: geth/jail/internal/fetch/Makefile (hunk 2 above)
Comment (divan) [reply geth/jail/internal/fetch/Makefile]: Oh wow, missed it, thanks!

**Pull request merged**